### PR TITLE
:white_check_mark: Adding tests for the child immunization structure map

### PR DESCRIPTION
**Pull request by dubdabasoduba**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes https://github.com/opensrp/fhir-resources/issues/91

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                {
+                  "viewType": "ROW",
+                  "children": [
+                    {
+                      "viewType": "PERSONAL_DATA",
+                      "personalDataItems": [
+                        {
+                          "label": {
+                            "viewType": "COMPOUND_TEXT",
+                            "primaryText": "82",
```
Comment (ellykits) [inline android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json:54]: Is this suppose to be static?

File: android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                            "primaryTextColor": "#000000"
+                          },
+                          "displayValue": {
+                            "viewType": "COMPOUND_TEXT",
+                            "primaryText": "Available",
```
Comment (ellykits) [inline android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json:59]: Should this be computed?

File: android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json (hunk 1 above)
Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json:54]: Nope we will update them once we have finalized the rules.

File: android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json (hunk 2 above)
Comment (dubdabasoduba) [reply android/quest/src/main/assets/configs/app/profiles/inventory_profile_config.json:59]: Nope, we will update them once we have finalized the rules.

**Review by ellykits — APPROVED** [review]

**Pull request merged**